docs/manual/customization.md

```diff
@@ -276,7 +276,7 @@ To run the controller in global mode, you will need to configure the following t
 
 You can build this kind of configuration easily from the sample deployment by changing the following things:
 
-* Delete the configuration for the `WATCH_NAMESPACE` variable
+* Delete the configuration for the `WATCH_NAMESPACE` variable or remove the command line option `-watch-namespace`.
```

Comment:
Since this is referring specifically to the sample deployment, I don't think we need to change this section of the docs until we change the sample deployment.